src/analysis/normed_space/basic.lean

```diff
@@ -86,6 +87,22 @@ noncomputable def semi_normed_group.of_core (α : Type*) [add_comm_group α] [ha
     calc ∥x - y∥ = ∥ -(y - x)∥ : by simp
              ... = ∥y - x∥ : by { rw [C.norm_neg] } }
 
+instance : normed_group punit :=
+{ norm := function.const _ 0,
+  dist_eq := λ _ _, rfl, }
+
+instance : normed_group ℤ :=
```

Comment:
We already have
```
src/analysis/normed_space/basic.lean
1132:instance : normed_comm_ring ℤ :=
```
and maybe something similar for `rat`?

Reply:
Yup, `normed_field rat` is already there. Removing both.